torch/_decomp/__init__.py

```diff
@@ -407,6 +407,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.sinc,
             aten.sinc_,
             aten.slice_backward,
+            aten.slice_scatter,
```

Comment:
Sorry but we need to keep `slice_scatter` in the ATen graph in order for it to work with the re-inplacing pass:
https://github.com/pytorch/pytorch/blob/c3de2cc1546378b2e96e0036aecded3e02a2c51d/torch/_inductor/fx_passes/reinplace.py#L32

This is so we can choose between the functional and mutating implementation based on some heuristics:
https://github.com/pytorch/pytorch/blob/c3de2cc1546378b2e96e0036aecded3e02a2c51d/torch/_inductor/fx_passes/reinplace.py#L170

Reply:
Can you remove it from the core decompositions instead. `slice_scatter` is a core ATen op, so it shouldn't be decomposed by default.